src/Neo/ProtocolSettings.cs

```diff
@@ -58,12 +58,16 @@ public record ProtocolSettings
         public string[] SeedList { get; init; }
 
         /// <summary>
-        /// Indicates the time in milliseconds between two blocks.
+        /// The milliseconds between two block in NEO.
+        /// For code that needs the accurate block generation time based on blockchain state, 
```

Comment:
And I won't call it accurate, because Policy's value can be much different from config's value. So just tell that this field should be considered as deprecated after Echidna.